### ISSUE TITLE
[Feature Request] - Login before start-page possible?

### ISSUE THREAD

**Issue by cjohnson46** [pawelmalak/flame#230]
Thanks for taking this project on from SUI and adding so many great features!  If you have a way of donating, I would be glad to.

I was wondering if a login page could be created before the start-page.  Using the settings page is a little cumbersome.  As of right now I'm using Nginx to force a login so no one can reach the start-page, but it would be nice to have a customized login and to not need to login again through settings page.

Thank you!

![Flame Login Idea_1](https://user-images.githubusercontent.com/80307224/143656691-dc26c4ef-d68b-4547-9d82-480d95655569.png)

**Comment by piraterx** [comment]
I think this will be good if they implement this.

**Comment by modem7** [comment]
@cjohnson46 what URL are you using to achieve that within nginx?

I'm looking to secure that login page w/ 2fa https://github.com/pawelmalak/flame/issues/222 but having some issues getting that to come up within the app.

**Comment by modem7** [comment]
> Sorry, the picture was just an example of a login page I created, nothing more.  I wish I could help you out with your problem but I'm no where near your level.

Ahhah! You're definitely adept at photo editing!

Hopefully both shall become a reality!

**Comment by EVIL-Teken** [comment]
Please add my voice to this feature request along with having the ability to change the password within the GUI.

**Comment by MrAlucardDante** [comment]
Why not using Authelia ? It's really easy to set up, enables login and 2FA in front of any container with labels !

**Comment by modem7** [comment]
> Why not using Authelia ? It's really easy to set up, enables login and 2FA in front of any container with labels !

Because that puts the entire service under, not just the login portal, which makes it bloody annoying when trying to actually use it as a start page.

**Comment by MrAlucardDante** [comment]
> > Why not using Authelia ? It's really easy to set up, enables login and 2FA in front of any container with labels !
> 
> Because that puts the entire service under, not just the login portal, which makes it bloody annoying when trying to actually use it as a start page, and offers nothing for public usage.

True, it's up to you how you want to implement security. Although I would rather not give access to everyone to all my apps links. but that's just me.

What could be nice is a way to disable the authentification entirely, this way if you use Authelia (or any auth provider), we don't have to log in twice. But I usually don't go the settings once my dashboard is set up, since new apps will automatically appear with docker labels.

**Comment by modem7** [comment]
> > > Why not using Authelia ? It's really easy to set up, enables login and 2FA in front of any container with labels !
> > 
> > Because that puts the entire service under, not just the login portal, which makes it bloody annoying when trying to actually use it as a start page, and offers nothing for public usage.
> 
> True, it's up to you how you want to implement security. Although I would rather not give access to everyone to all my apps links. but that's just me.
> 
> What could be nice is a way to disable the authentification entirely, this way if you use Authelia (or any auth provider), we don't have to log in twice. But I usually don't go the settings once my dashboard is set up, since new apps will automatically appear with docker labels.

Flame can happily give a public view without all of your services and links, this is useful for request portals etc. 

You are also able to hide different services.

The only thing required is a more secure login portal tbf.
